examples/k8s.yaml

```diff
@@ -197,7 +200,7 @@ spec:
             cpu: 200m
             memory: 400Mi
         ports:
```

Comment:
I'm not super familiar with k8s so ignore me if I'm wrong, but I think you should add the zpages port here since you enabled it for the `otel-collector`

Reply:
correct